api/net/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN dotnet publish "TNO.API.csproj" -c "$BUILD_CONFIGURATION" -o /app/publish
 FROM base AS final
 
 RUN apt-get update && apt-get -y upgrade
-RUN apt -y install curl libc6-dev libgdiplus ffmpeg
+RUN apt -y install curl libc6-dev
```

Comment:
Why are you removing these dependencies from the API?

Reply:
I copied that line from the Dockerfile for the API container. I removed ffmpeg because it's only required in the capture and clip containers (so far). Looks like libgdiplus is a graphics package, which I didn't think was required in the transcription service. What are the plans for using it in the API?

Reply:
This is the API Dockerfile, I believe your video editing features require these libraries.  So I don't know why you'd remove it in this PR.

Reply:
Oops...didn't notice which Dockerfile that was. I've replaced them. Still not sure what libgdiplus is used for by the API...I didn't add it.